internal/app/ui/menu/menu.go

```diff
@@ -193,6 +194,10 @@ func (m *Menu) Process() {
 				Icon(icon.Search).
 				Enabled(m.app.HasActiveMap()).
 				Shortcut(platform.KeyModName(), "F"),
+			w.MenuItem("Go to Coords", m.app.DoOpenJumpWindow).
+				Icon(icon.Shrink).
+				Enabled(m.app.HasActiveMap()).
+				Shortcut(platform.KeyModName(), "G"),
```

Comment:
To add shortcuts you also need to modify file https://github.com/SpaiR/StrongDMM/blob/main/internal/app/ui/menu/shortcut.go
Otherwise nothing will be handles

Reply:
done